apache-rat-plugin/pom.xml

```diff
@@ -157,6 +157,7 @@
         <plugin>
           <groupId>org.codehaus.mojo</groupId>
           <artifactId>build-helper-maven-plugin</artifactId>
+          <version>3.2.0</version>
```

Comment:
This seems redundant as we define the version in the main pom under pluginManagement and use 3.6.0 in master branch.

Reply:
Fixed.